### PR TITLE
Cleanup the GDD

### PR DESCRIPTION
**Pull request by hugglesfox**
Cleanup formatting to make it more consistent as well as fixed spelling mistakes. Also added Linux as a compatible operating system.

### CODE REVIEW

**Review by ghost — APPROVED** [review]
looks good

File: GDD.md [hunk 1]
```diff
@@ -42,61 +42,61 @@
...
 
 ## Platform:
 
--   PC
+- PC
 
 ## Development Environment:
 
--		unity
+-	Unity3D
```
Comment (Aprilla999) [inline GDD.md]: Change to python/pyglit

File: GDD.md [hunk 2]
```diff
@@ -42,61 +42,61 @@
...
 
 ## System Requirements:
 
-- 	windows 10
+- x86_64 Windows 10 or Linux PC
```
Comment (Aprilla999) [inline GDD.md]: 'and' in place of 'or'

**Review by Aprilla999 — CHANGES_REQUESTED** [review]
Still work to be done

**Review by Aprilla999 — APPROVED** [review]

**Pull request merged**